### DISCUSSION TITLE
/cn/2019/09/international-autonomous-greenhouse-challenge/

### DISCUSSION THREAD

**Discussion by leovan** [leovan/leovan.me#16] [Comments]
国际智慧温室种植挑战赛 (International Autonomous Greenhouse Challenge) 
 https://leovan.me/cn/2019/09/international-autonomous-greenhouse-challenge/

**Comment by leovan** [comment]
```header
由 Kis Chang 于 2021-09-09 12:09:54 写道：
````
请教一下，哪里能够获得Kaspro 温室模型的相关资料？

**Reply by leovan** [reply]
```header
于 2021-09-18 10:00:29 写道：
````
已经不在做农业方向了，不过之前理解我们的目的就是要做一个模型去拟合他们的
Kaspro 参数，Kaspro
模型之前应该在网上搜过一些资料，也不是太多，我理解算是瓦大的核心吧，有很多细节没有对外公布。

**Comment by shibonW** [comment]
博主您好，我是一名农林类高校的准大四计算机专业的本科生。我的毕业设计想设计与温室控制相关的算法，但是缺乏方向，您可以简要的向我列举几个从入门到进阶的书籍或学习路径嘛？非常感谢！

**Reply by leovan** [reply]
抱歉，在这方面研究甚浅，当年也仅仅是参加了一下这个比赛，所以在这方面给不到建议。

**Comment by dxp397** [comment]
我们正在做温室环境控制系统的国产化研究，可以聊一下吗？可有偿

**Reply by leovan** [reply]
抱歉，在这方面研究甚浅，当年也仅仅是参加了一下这个比赛，现在已经不在做农业方向了。

**Reply by duirixuanyan** [reply]
同样做这个方向的研究，姓赵，15114136169，感兴趣可以联系。
